### ISSUE TITLE
[Bug]: failing to get alerts list using Go SDK

### ISSUE THREAD

**Issue by wasilak** [coralogix/coralogix-management-sdk#87]
### Expected Result

I am expecting to be able to list alerts :)

### Actual Result

Hey guys! please advise :) I'm getting a proto related error when using `func (a cxsdk.AlertsClient) List(ctx context.Context, req *v3.ListAlertDefsRequest) (*v3.ListAlertDefsResponse, error)`, see below.

```
rpc error: code = Internal desc = grpc: failed to unmarshal the received message: proto: cannot parse invalid wire-format data
```

This is the code I'm using:

```go
package coralogix

import (
	"log/slog"

	cxsdk "github.com/coralogix/coralogix-management-sdk/go"
)

type ConnectionAlerts struct {
	AlertsClient *cxsdk.AlertsClient
}

func (c *Coralogix) setupConnectionAlerts() ConnectionAlerts {
	authContext := cxsdk.NewAuthContext(c.connectionParams.TeamLevelAPIKey, c.connectionParams.UserLevelAPIKey)
	creator := cxsdk.NewCallPropertiesCreator(c.connectionParams.Region, authContext)

	return ConnectionAlerts{
		AlertsClient: cxsdk.NewAlertsClient(creator),
	}
}

func (c *Coralogix) GetAlertsMetrics() error {
	c.connectionAlerts = c.setupConnectionAlerts()

	err := c.generatAlertsListMetric()
	if err != nil {
		return err
	}

	return nil
}

func (c *Coralogix) generatAlertsListMetric() error {
	alertDefReq := &cxsdk.ListAlertDefsRequest{}
	alerts, err := c.connectionAlerts.AlertsClient.List(c.CTX, alertDefReq) // <------  here the error happens 
	if err != nil {
		return err
	}

	for _, alert := range alerts.AlertDefs {
		slog.DebugContext(c.CTX, "Alert", "name", alert.GetId(), "description", alert.AlertDefProperties.GetDescription())
	}

	return nil
}
```

### Relevant log output

```shell
rpc error: code = Internal desc = grpc: failed to unmarshal the received message: proto: cannot parse invalid wire-format data
```

**Comment by celaus** [comment]
Hi - what version of the SDK are you using?

**Comment by wasilak** [comment]
Hey! :) 

I'm on github.com/coralogix/coralogix-management-sdk/go v0.0.0-20240822133107-2f630415357f
it was latest at the time, so around Nov 13, 2024

**Comment by celaus** [comment]
Fixed with #102

**Comment by wasilak** [comment]
Amazing, thank you!
